aggregate.go

```diff
@@ -1,69 +1,85 @@
 package dogma
 
-// AggregateMessageHandler is an interface for applying changes to an aggregate
-// via command messages.
+// AggregateMessageHandler is an interface implemented by the application and
+// used by the engine to cause changes to an aggregate via command messages.
```

Comment:
I feel like this could use a comma after "used by the engine"?

Reply:
Or after "implemented by the application", as was done further down on other types.